.ci/ci_crio_entry_point.sh

```diff
@@ -126,6 +126,7 @@ golang_version="1.18.1"
 yq w -i versions.yaml languages.golang.meta.newest-version "${golang_version}"
 
 critools_version="${branch_release_number}.0"
+[ ${critools_version} == "1.24.0" ] && critools_version="1.24.2"
```

Comment:
Isn't critools_version was changed to 1.24.2 already?

Reply:
Nops, here we set the critools version related to the CRI-O branch.
If using CRI-O 1.24.x, we need to ensure we use critools 1.24.2+